### ISSUE TITLE
New installation challenge PHP Version

### ISSUE THREAD

**Issue by cechuchu57** [webkul/hotelcommerce#1160]
Please advise on a way forward. I am trying to install Qloapps and I get the below

![image](https://github.com/user-attachments/assets/97df25da-2897-434b-b66f-a3caca739f16)

I greatly I appreciate any pointers

**Comment by Deeksharanaba** [comment]
Hi @cechuchu57 , 

The error that you are receiving while installing QloApps is due to PHP version incompatibility. **The required PHP version on your server to install QloApps should be between 5.6 to 7.4**

Kindly connect with the server provider and request the server provider to change the PHP version to 7.4. This will resolve your issue and you will be able to install QloApps. 

Please refer to our requirement document to know the requirements of the server configuration for the installation of QloApps. 
QloApps installation requirement doc: https://devdocs.qloapps.com/introduction/requirements.html

**Comment by github-luckow-org** [comment]
ah. no way. https://www.php.net/supported-versions.php
7.4 is dead since end of 2022

**Comment by sumitwebkul** [comment]
We understand your concern about upgrading the PHP Version in QloApps.

I want to inform you that the PHP and MySQL version upgrade is already under development. We will switch the QloApps PHP and MySQL versions to 8. x soon.

We can not provide you with a tentative release date for it, but we will surely inform you once the QloApps with upgraded PHP and MySQL versions are released.

**Comment by chadek** [comment]
@sumitwebkul any update on this? Do you have any approximated idea on when such release will be publish? Thanks in advance

**Comment by carlo-ztl** [comment]
I agree on the request for a timeline for this fix. This is a security concern because PHP7 reached EOL in November 2022. Current PHP supported versions are 8.1+.

**Comment by AradhanaSingh63** [comment]
Hi @carlo-ztl ,  @cechuchu57, @github-luckow-org , @chadek 

QloApps also release with the latest php version and mysql version , all libraries are update as per the php version.
